src/vivarium_public_health/risks/base_risk.py

```diff
@@ -36,17 +43,24 @@ def setup(self, builder):
 
     def _get_distribution(self, builder):
         """A wrapper to isolate builder from setup"""
+        kwargs = {}
+        if builder.configuration[self._risk]['exposure'] != 'data':
+            if builder.configuration[self._risk]['distribution'] != "dichotomous":
+                raise ValueError("A Dummy Risk must have a dichotomous distribution.")
```

Comment:
Raise a not implemented error here.  We may support more complicated things later.